project/Dependencies.scala

```diff
@@ -19,14 +19,14 @@ object Dependencies {
     val SbtScala = Seq(Scala212, Scala210)
 
     // If you update the version of Play, you probably need to update the other Play* variables.
-    val Play             = "2.8.8" // sync with docs/build.sbt
+    val Play             = "2.8.11" // sync with docs/build.sbt
     val PlayJson         = "2.9.1"
-    val PlayStandaloneWs = "2.1.3"
+    val PlayStandaloneWs = "2.1.6"
     val Twirl            = "1.5.1" // sync with docs/project/plugins.sbt
     val PlayFileWatch    = "1.1.14"
 
-    val Akka: String = sys.props.getOrElse("lagom.build.akka.version", "2.6.14") // sync with docs/build.sbt
-    val AkkaHttp     = "10.1.14"
+    val Akka: String = sys.props.getOrElse("lagom.build.akka.version", "2.6.17") // sync with docs/build.sbt
+    val AkkaHttp     = "10.1.15"
```

Comment:
Don't we want to update Akka HTTP to 10.2 as hinted for users in #3319? 🤔 
Or should we wait for Play 2.9 released and then update the version of Play and Akka HTTP together?

Reply:
I think we will play on the safe side if we don't ship with it. 
The security fixes have been backported to 10.1.15, so in principle as long as Play 2.8 is being used, it's wiser to keep the version aligned.